RELEASE-NOTES.txt

```diff
@@ -1,6 +1,12 @@
+23.5
+-----
+
```

Comment:
I think we don't need to define the next version, AFAIK this change is made as part of the release process of the app.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/21652/commits/946e73f7bb5127ca40c108b89767bf56d52d5567.